src/portal/views/Dao.vue

```diff
@@ -34,90 +29,88 @@
         </v-card-text>
 
-        <v-container
-          fluid
-          class="d-flex justify-space-between my-5"
-        >
+        <v-container fluid class="d-flex justify-space-between my-5">
           <v-btn
-            color="green"
-            class=" "
-            :width="`${proposal.ayes.length *100 + 100}`"
+            color="primary"
+            class=""
+            width="100px"
             @click="openVoteDialog(proposal.hash, true)"
             :loading="loadingVote"
-          >Yes
-            <v-divider
-              class="mx-3"
-              vertical
-            />{{proposal.ayes.length}}
+            >Yes <v-divider class="mx-3" vertical />{{ proposal.ayes.length }}
           </v-btn>
 
-          <div class="d-flex align-center  text-center threshold">
+          <div class="d-flex align-center text-center threshold">
             <v-divider vertical />
-            <span>Threshold: <br>{{(proposal.nayes.length + proposal.ayes.length)}}/{{proposal.threshold}}
-
+            <span
+              >Threshold: <br />{{
+                proposal.nayes.length + proposal.ayes.length
+              }}/{{ proposal.threshold }}
             </span>
             <v-divider vertical />
-
           </div>
           <v-btn
-            color="red"
-            :width="`${proposal.nayes.length *100 + 100}`"
+            color="grey lighten-2 black--text"
+            width="100px"
             @click="openVoteDialog(proposal.hash, false)"
             :loading="loadingVote"
-          >No
-            <v-divider
-              class="mx-3"
-              vertical
-            />{{proposal.nayes.length}}
+            >No <v-divider class="mx-3" vertical />{{ proposal.nayes.length }}
           </v-btn>
-
         </v-container>
-        <v-container>
+        <v-container
+          v-if="proposal.ayesProgress > 0 || proposal.nayesProgress > 0"
+        >
           <v-row justify="center">
-
-            <v-col
-              :sm="Math.round(proposal.ayesProgress*12/100)===12? 11: proposal.ayesProgress*12/100"
+            <v-progress-linear
+              :value="proposal.ayesProgress"
               height="25"
-              style="background-color: green;"
+              :style="{
+                width: proposal.ayesProgress + '%',
+                marginRight: 'auto',
+                backgroundColor: '#1982b1',
+                color: '#fff',
+              }"
             >
-              <span>{{ proposal.ayesProgress  }}%</span>
-            </v-col>
-            <v-col
+              <template>
+                <strong>{{ proposal.ayesProgress }}%</strong>
+              </template>
+            </v-progress-linear>
+            <v-progress-linear
+              :value="proposal.nayesProgress"
+              color="grey lighten-2"
               height="25"
-              style="background-color: red;"
-              :sm="Math.round(proposal.nayesProgress*12/100)===12? 11: proposal.nayesProgress*12/100"
```

Comment:
please leave the calculation here as well to allow visualization of nayes progress as such
![Screenshot from 2022-06-28 13-49-51](https://user-images.githubusercontent.com/56790126/176171722-cc947e8a-110f-48ae-9249-d326993f19b6.png)